include/nccl_ofi_sendrecv.h

```diff
@@ -58,7 +59,8 @@ typedef struct nccl_net_ofi_sendrecv_send_comm {
 	fi_addr_t local_ep_addr;
 	struct fid_ep *local_ep;
 
-	nccl_ofi_connection_info_t *conn_info;
+	/* connecting peer information (nccl_ofi_connection_info_t) */
+	nccl_ofi_freelist_elem_t *conn_info;
```

Comment:
Not pertaining to the MR_LOCAL change itself, so separate commit? But 👍 for moving this into a fl too.

Reply:
this is part of the MR_LOCAL change.  These are things going on the wire, and we either need to have them in a MR-backed freelist or explicitly register.  I went with the MR-backed freelist approach, and this is the fallout of that.